src/test/java/seedu/address/model/person/DeadlineNoteTest.java

```diff
@@ -48,21 +46,24 @@ public void convertSuccessfulDate() {
 
     @Test
     public void equals() {
-        Note deadlineNote = new DeadlineNote("kind cats", "2025-10-10");
+        Note deadlineNote1 = new DeadlineNote("kind cats", "2025-10-10");
```

Comment:
variable names should not contain numbers!

See [here](https://nus-cs2103-ay2324s2.github.io/website/schedule/week4/topics.html#w4-6-code-quality-naming)